lti_consumer/lti_xblock.py

```diff
@@ -1085,6 +1085,19 @@ def lti_1p3_launch_callback(self, request, suffix=''):  # pylint: disable=unused
             # This is optional though
             lti_consumer.set_launch_presentation_claim('iframe')
 
+            # Set context claim
+            # This is optional
+            context_title = " | ".join([
+                self.course.display_name_with_default,
+                self.course.display_org_with_default
+            ])
+            lti_consumer.set_context_claim(
+                self.context_id,
+                context_types=["course_section"],
```

Comment:
@pcockwell Can you change this to `course_offering`? 
Otherwise the context data won't match the actual key being passed (we're passing course information, but using the course section type?)

Reply:
Done in next commit